tests/PLAYWRIGHT_GUIDE.md

```
@@ -0,0 +1,468 @@
+      // actions
+    });
+
+    await test.step("Step 2: Verify result", async () => {
+      // assertions
+    });
+  });
+});
+```
+
+## Authentication
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)